source/renderer/app/api/ada/index.js

```diff
@@ -267,21 +267,36 @@ export default class AdaApi {
     const { walletId, skip, limit } = request;
 
     const accounts: AdaAccounts = await getAdaWalletAccounts(this.config, { walletId });
-    const accountIndex = accounts[0].index;
-    const page = skip === 0 ? 1 : (skip / limit) + 1;
-    const perPage = limit > 50 ? 50 : limit;
 
     const params = {
-      accountIndex,
-      page,
-      per_page: perPage,
+      accountIndex: accounts[0].index,
+      page: skip === 0 ? 1 : (skip / limit) + 1,
+      per_page: limit > 50 ? 50 : limit,
       wallet_id: walletId,
       sort_by: 'DES[created_at]',
     };
 
+    const pageLimit = Math.ceil(limit / 50);
```

Comment:
@daniloprates we need to use `params.per_page` instead of `50` here in order to cover the load-5-transactions request for the wallet summary screen. In that case it should be:
```
const pageLimit = Math.ceil(5 / 5); // which equals 1
```

Reply:
@daniloprates I've added this fix: https://github.com/input-output-hk/daedalus/pull/1078/commits/ff356163643c154c3e922c03737253953067a105

Reply:
that's right! thanks for the fix!